config.sample.yaml

```diff
@@ -59,6 +59,7 @@ ircService:
       # Whether or not IRC server is using a self-signed cert or not providing CA Chain
       sslselfsign: false
       # Whether to allow expired certs when connecting to the IRC server.
+      # This protects explitially against expired certs, not invalid ones.
```

Comment:
this probably should be in that other PR that was merged ;)

Reply:
This was the sentence I wrote before I changed my mind :p